packages/amplify-category-auth/provider-utils/awscloudformation/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -367,7 +369,7 @@ Resources:
             - ' }'
             - '};'
       Handler: index.handler
-      Runtime: nodejs10.x
```

Comment:
Did you try pushing this template and testing it? 
I believe Cloudformation doesn't support in-line lambda's with Node 12.x runtime.

Reply:
CloudFormation does support Lambda Node 12.x runtime. [Here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-runtime) is the documentation.

Reply:
This is for the Lambda functions - not for the in-line custom resources which you define in your cloudformation file. Have you tried pushing this template successfully?

Reply:
What do you mean "for the Lambda functions"? If you mean I was able to create those Lambda functions with Node 12.x runtime, like I said, CloudFormation supports Lambda Node 12.x runtime, so the answer is yes.
![image](https://user-images.githubusercontent.com/22212362/75481306-aeecc200-5957-11ea-8c88-73486b22f72b.png)

Not for ```OpenIdLambda``` because I have not figured out how to create that one yet.